data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -91,11 +91,23 @@ def format_offices_data(data: dict) -> list[dict]:
     offices_data = data['offices']
     formatted_offices = []
 
+    # Map office type codes to office type names
+    # TODO: support other office types
+    office_type_mapping = {
+        'RC': 'recordsOffice',
+        'RG': 'registeredOffice',
+        'DS': 'dcrOffice',
```

Comment:
```suggestion
        'DS': 'custodialOffice',
```

ref:
https://github.com/bcgov/lear/blob/main/colin-api/src/colin_api/models/office.py#L34
https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py#L75

Reply:
Ah that simplifies things! I'll update and remove the migration script.